LenovoLegionToolkit.Lib.Automation/Pipeline/Triggers/IAutomationPipelineTrigger.cs

```diff
@@ -79,3 +79,9 @@ public interface IWiFiConnectedPipelineTrigger : IAutomationPipelineTrigger
 }
 
 public interface IWiFiDisconnectedPipelineTrigger : IDisallowDuplicatesAutomationPipelineTrigger { }
+
+public interface IPeriodicAutomationPipelineTrigger : IAutomationPipelineTrigger 
+{
+    public int? PeriodMinutes { get; }
```

Comment:
I would consider using `TimeSpan` to store the value - will be more future proof.